charts/latest/blob-csi-driver/Chart.yaml

```diff
@@ -2,4 +2,4 @@ apiVersion: v1
 appVersion: latest
 description: Azure Blob Storage CSI driver
 name: blob-csi-driver
-version: v1.2.0
+version: v1.2.1
```

Comment:
pls keep `v1.2.0`, only need to change `rbac-csi-blob-node.yaml`

Reply:
Updated.